First draft

potomak opened the pull request.

potomak committed:
First draft

potomak committed:
Update subtitle

Let's be more generic, Open Source is not just software.

potomak committed:
Update introduction section

Make it clearer that the app will enable crowdfunding on the Ethereum
blockchain.

bugant commented on index.md:
```diff
@@ -5,4 +5,83 @@ title: issuehunter
+It makes it easier for supporters to funds projects they love and for Open
+Source contributors to collect rewards for their work.
+
+The dapp is composed by three main elements:
```
I'd change this to: The decentralized app (dapp)

potomak replied:
Done.

bugant commented on index.md:
```diff
@@ -5,4 +5,83 @@ title: issuehunter
+* a set of smart contracts deployed on the Ethereum blockchain
+* a service that helps verifying software patches submitted
+* a web based directory to discover funded issues that are still waiting to be
+  resolved
```
Do we also have integration (an automatic comment or something like that) with github (and others to come later) to show directly on the issue page that there is an active bounty campaign on Issuehunter?

potomak replied:
Yes, that's correct, I forgot to mention it!

bugant commented on index.md:
```diff
@@ -5,4 +5,83 @@ title: issuehunter
+
+Contributors add funds to any unresolved issue on GitHub. The act of adding
+funds to an open issue triggers a smart contract function call and a transaction
+of Ethereum between the funder and the contract that is controlling the issue's
```
and a transaction on the Ethereum blockchain with a fund transfer between the funder and the contract...

potomak replied:
Done.

bugant commented on index.md:
```diff
@@ -5,4 +5,83 @@ title: issuehunter
+funds.
+
+You can add funds to any issue on GitHub. If the issue was already funded by
+other contributors, your funds will add up to the total funds of the issue.
```
Otherwise, a fresh new contract will be created to collect funds to promote and sponsor the solution of the issue.

potomak replied:
Done.

dalpo commented on index.md:
```diff
@@ -5,4 +5,83 @@ title: issuehunter
 
-What a wonderful project!
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
```
I'd put a link to the Ethereum website (https://www.ethereum.org/)

potomak replied:
I'll put a link to the Ethereum website, but I wouldn't like to do it on the first sentence of the introduction to better focus the attention on the content.

dalpo replied:
:+1:

potomak committed:
Address review comments

As a result "Step 1" should be a little bit clearer now.

potomak committed:
Update introduction

Make the separation between web services and dapp components clearer
and add a missing component to notify watchers that there is an active
bounty campaign on issuehunter.

See https://github.com/issuehunter/issuehunter.github.io/pull/1#discussion_r131326168
